src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -345,7 +345,25 @@ private HttpMessageHandler SetupHandlerChain()
                 return Task.FromException<HttpResponseMessage>(error);
             }
 
-            return handler.SendAsync(request, cancellationToken);
+            if (HttpTelemetry.IsEnabled && request.RequestUri != null)
```

Comment:
Is there a case where `RequestUri` would be `null` here?

Reply:
Probably not but request.RequestUri is a property that can return null in theory. I was getting nullable compile errors without this check. 

@stephentoub Can you comment on this?

Reply:
It can be null here, e.g. it'll be null here if you do:
```C#
var invoker = new HttpMessageInvoker(new SocketsHttpHandler());
await invoker.SendAsync(new HttpRequestMessage(), default);
```

Reply:
(I don't know why ValidateAndNormalizeRequest doesn't produce an exception if the Uri is null.  If we wanted to change that so that it did and we'd never get here with a null Uri, then we could Debug.Assert(request.RequestUri != null) instead.)